pkg/scheduler/framework/plugins/interpodaffinity/interpod_affinity.go

```diff
@@ -55,39 +59,303 @@ var _ framework.FilterPlugin = &InterPodAffinity{}
 var _ framework.PostFilterPlugin = &InterPodAffinity{}
 var _ framework.ScorePlugin = &InterPodAffinity{}
 
-const (
-	// Name is the name of the plugin used in the plugin registry and configurations.
-	Name = "InterPodAffinity"
-
-	// preFilterStateKey is the key in CycleState to InterPodAffinity pre-computed data for Filtering.
-	// Using the name of the plugin will likely help us avoid collisions with other plugins.
-	preFilterStateKey = "PreFilter" + Name
+// InterPodAffinity is a plugin that checks inter pod affinity
```

Comment:
now I feel like this file is too big. But the plugin lives in a folder, so we can do something like:

```
interpodaffinity/
  plugin.go
  filtering.go
  scoring.go
```

WDYT?

Reply:
sg, I was thinking the same thing.

Reply:
done.